src/Controller/ActivityController.php

```diff
@@ -488,16 +482,49 @@ public function applyForActivity(Activity $activity, ActivityUserRepository $act
     }
 
     /**
-     *Apply for an Activity.
+     *Get a list of all applicants.
      * @Rest\Get("/{id}/applicants")
+     * @SWG\Get(
+     *     tags={"Activity"},
+     *     summary="Get a list of all applicants",
+     *     description="Get a list of all applicants",
+     *     operationId="getApplicants",
+     *     produces={"application/json"},
+     *     @SWG\Parameter(
+     *     description="ID of Activity",
+     *     in="path",
+     *     name="id",
+     *     required=true,
+     *     type="integer",
+     * )
+     * )
+     * @SWG\Response(
+     *     response=200,
+     *     description="Successfull operation!",
+     *     @SWG\Schema(
+     *     type="array",
+     *     @Model(type=User::class, groups={"UserDetail"}),
```

Comment:
`UserList` should be used as there's no need to show all the details about the user. Make sure it exists.